style.css

```diff
@@ -0,0 +1,268 @@
+* {box-sizing: border-box;
+    padding: 0;
+    margin: 0;}
+ body{
+        background-color: rgb(206, 224, 230);
+        color: rgb(17, 27, 28);
+        font-family: Georgia, 'Times New Roman', Times, serif;}
+        
+a{color: rgb(17,27,28);
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  font-weight: bolder;}
+
+img{max-width: 100%}
+
+header 
+    {padding: 1rem .5rem;
+    margin: 1rem 1rem 0 1rem;
+    background-color: rgb(71, 134, 146);
+  }
+
+  @-o-viewport {
+    width: 320px;
+    zoom: 0.5;
+  }
+
+main
+    {padding: 1rem 1rem 2rem;
+    display: flex;
+    justify-content: space-between;
+    align-items: center}
+
+footer{padding: 1rem;
+    background-color: rgb(34, 64, 70);
+    color: rgb(206, 224, 230);}
+    
+.main-menu
+    {display: flex;
+    justify-content: space-between;
+    position: relative}
+
+@media (min-width:599px) { 
+  .btn {
+  list-style-type: none;
+  margin: 0;
+      }
+.drops {
+  display: inline;
+  width: 2in;
+  float: right;
```

Comment:
let's try to avoid using floats. this is resulting in a different order visually from the HTML structure which is an accessibility issue. use flexbox instead.